tests/schema/schema.go

```diff
@@ -0,0 +1,22 @@
+package schema
+
+import (
+	clientv3 "go.etcd.io/etcd/client/v3"
+)
+
+type EndpointStatus struct {
```

Comment:
Do we need endpoint struct at all? Looks like you have them just to parse json response from etcdctl. However if they are not used anywhere else maybe anonymous struct would be good enough?

Reply:
Thanks review. Done!